cmd/grafiti/main.go

```diff
@@ -89,23 +168,26 @@ func initConfig() {
 	viper.SetDefault("maxNumRequestRetries", 8)
 
 	// If a config file is found, read in its data
-	if err := viper.ReadInConfig(); err == nil {
-		logrus.Info("Using config file: ", viper.ConfigFileUsed())
+	if err := viper.ReadInConfig(); err != nil {
+		logger.Errorln("read config file:", err)
+	} else {
+		logger.Infoln("Using config file:", viper.ConfigFileUsed())
 		return
 	}
 
 	// No config file found so configure grafiti with a dummy config file (Reader)
 	// and environment variables
 	viper.SetConfigType("toml")
-	if err := viper.ReadConfig(bytes.NewBuffer([]byte(""))); err == nil {
-		logrus.Info("Using environment variables to configure grafiti.")
+	if err := viper.ReadConfig(bytes.NewBuffer([]byte(""))); err != nil {
+		logger.Errorln("read dummy config file:", err)
+	} else {
+		logger.Info("Using environment variables to configure grafiti.")
 		return
 	}
-
 }
 
 func main() {
 	if err := RootCmd.Execute(); err != nil {
```

Comment:
Whenever a command registered as a RunE exits, cobra will print help text. I've found this to be an annoying experience when an error unrelated to help information occurs. With both error functions we can control error text in a relatively simple manner.